src/ostorlab/cli/ci_scan/run/assets/android.py

```diff
@@ -0,0 +1,44 @@
+"""Asset of type .APK package file.
```

Comment:
I would recommend keeping the same assets we have on scan run. Meaning android_apk, android_aab, etc.

Reply:
This also means adding an asset package_name and bundle_id